### ISSUE TITLE
Computer transfering between entities - licenses issues

### ISSUE THREAD

**Issue by felipebart10** [glpi-project/glpi#9532]
I have an issue that I can't figure which parameter is causing or if it is a bug. It is happening both on 9.4.6 and 9.5.5, with all plugins disabled. When I transfer computers between entities, some licenses are being thrown at the bin and the license number is incremented by 1 (this doesn't make sense because some OEM licenses can't have more than 1). Every time I transfer a PC I have to restore the deleted licenses and change the 'number' field to 1 again. We've even started changing the location directly on the database, but this isn't the greatest solution as human error can happen. I will put some screenshots for you guys understand what is happening. I've actually did a little research, found some people with similar issues but couldn't find the solution. Thank you all for your attention!

**Page(s) URL**
/front/transfer.action.php

**To reproduce**
1. Go to Assets > Computers
2. Mark a computer with softwares/licenses and on 'Actions' choose 'Add to transfer list'
3. Choose a transfer template and an entity to transfer the PC
4. Go to Management>Licenses and see the trashbin with some licenses that belongs to the transfered PC.
5. On each license, see the 'Number' field to see that it was incremented by 1. 

**Expected behavior**
The transfer routine shouldn't put licenses on the trashbin. It was supposed to only transfer the computer withour affecting the software's licenses.

**Logs**
As no error messages happen, I've seen the debug and identified these queries modifying licenses registers:

```
UPDATE `glpi_softwarelicenses` SET `is_deleted` = '1', `date_mod` = '2021-09-10 10:41:17' WHERE `id` = '409'
UPDATE `glpi_softwarelicenses` SET `number` = '2' WHERE `id` = '409'

UPDATE `glpi_softwarelicenses` SET `is_deleted` = '1', `date_mod` = '2021-09-10 10:50:35' WHERE `id` = '410'
UPDATE `glpi_softwarelicenses` SET `number` = '2' WHERE `id` = '410'
```

```
GLPI 9.4.6 ( => /var/www/html/glpi)
Installation mode: TARBALL
```

Images:

Before transfer:
![Inkedbefore_LI](https://user-images.githubusercontent.com/64552706/132874755-e615e6fe-68b3-4b6e-9db1-9835174a3d79.jpg)

After transfer (note that the register is on the trashbin):
![Inkedafter_LI](https://user-images.githubusercontent.com/64552706/132874815-b1d7c970-3ff9-4a7c-a48b-a28e8c123f76.jpg)

**Comment by felipebart10** [comment]
Ok, I think I recognised what is happening. When 'Number' field is 1, the license gets sent to the trashbin. This seems to be happening just because this reason. I tested with different licenses types and the problem occurred with them all only if 'number' is 1. I tested this on both 9.4.6 and 9.5.5. Note that the changelog show when 'number' is 1, the register is deleted but not restored, but when it is 2, the register is updated two times, first decreasing and then increasing the 'number' by 1.

![image](https://user-images.githubusercontent.com/64552706/132897671-56c21e01-75a0-49f3-bc36-720c6fd2f75b.png)
